packages/rich-text/src/plugins/List/utils.ts

```diff
@@ -37,8 +37,20 @@ export const isNonEmptyListItem = (editor: PlateEditor, [, path]: NodeEntry) =>
   return listItemChildren.length !== 0;
 };
 
+export const firstNodeIsNotList = (_editor: PlateEditor, [node]: NodeEntry<CustomElement>) => {
+  const firstNode = node.children?.[0];
+  return !Text.isText(firstNode) && !isList(firstNode);
```

Comment:
`isList` doesn't expect the node to be `undefined` which can happen in this case.

Reply:
Updated :) tks